src/web/src/app/space/detail/component.ts

```diff
@@ -0,0 +1,277 @@
+import { Component, OnInit, ViewChild, Inject, Injectable } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { MatPaginator, PageEvent } from '@angular/material/paginator';
+import { DOCUMENT } from '@angular/common';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+
+import { Space } from '../model';
+import { SpaceInvitation } from '../model';
+import { SpaceLink } from '../model';
+import { SpaceUser } from '../model';
+
+import { SpaceService } from '../service';
+import { UserService } from '../../user/service';
+import { RequestWatcherService } from '../../request-watcher/service';
+
+@Component({
+	selector: 'app-space-detail',
+	templateUrl: './component.html',
+	styleUrls: ['./component.css']
+})
+export class SpaceDetailComponent implements OnInit {
+
+	invitationForm: FormGroup;
+	isInvitationFormHidden: boolean = true;
+
+	linkForm: FormGroup;
+	isLinkFormHidden: boolean = true;
+
+	currentSpace: Space;
+	currentUserId: string = "2";
+	currentSpaceId;
+
+	linksURL: string = "/space/link/";
+
+	invitations: SpaceInvitation[] = [];
+	links: SpaceLink[] = [];
+	users: SpaceUser[] = [];
+
+	invitationsPageSize: number = 7;
+	linksPageSize: number = 7;
+	usersPageSize: number = 10;
```

Comment:
Может сделаем под это общую структуру? Так будет проще сохранять.
```suggestion
	pageSize = {
		invitations: 7,
		links: 7,
		users: 10,
	};
```

Reply:
Готово.